app/views/games/categories/_title.slim

```diff
@@ -1,4 +1,4 @@
-.card.mb-3
+.card.mb-3.bg-darkish
```

Comment:
Is every single card going to have `.bg-darkish`?  If so is there a way to just set this once so we don't need the remember to add it everytime?

Reply:
I thought I'd have to do some weird SASS `@extend`ing that wouldn't be friendly to upgrades, but this prompted me to do some experimenting and I found an apparently undocumented variable that works great!